package.json

```diff
@@ -23,9 +23,9 @@
     "lint:fix": "prettier \"src/**/*.ts\" --write && tslint -c tslint.json -p tsconfig.json --fix",
     "prepublishOnly": "yarn build",
     "publish-module": "cognite-release-manager publish-module",
-    "test": "yarn jest",
+    "test": "NODE_PATH=src/ yarn jest",
     "test:watch": "yarn jest --watchAll",
-    "storybook": "start-storybook -p 6006"
+    "storybook": "NODE_PATH=./src start-storybook -p 6006"
```

Comment:
Same here

Reply:
I removed this but I guess it was overwritten... will change again